components/BudgetItemsList.js

```diff
@@ -314,15 +315,17 @@ const BudgetItem = ({ item, isInverted, isCompact, canDownloadInvoice, intl }) =
             mt={[2, 0]}
           >
             <Flex alignItems="center" flexWrap="wrap">
-              {description ? (
-                <P color="black.900" fontWeight="600" fontSize="Paragraph" title={description}>
-                  {truncate(description, { length: 60 })}
-                </P>
-              ) : (
-                <P color="black.500" fontStyle="italic" fontWeight="600" fontSize="Paragraph">
-                  <FormattedMessage id="BudgetItemsList.NoDescription" defaultMessage="No description provided" />
-                </P>
-              )}
+              <Link route={`/${collective.slug}/expenses/${item.id}`} title={description}>
```

Comment:
`item` is not necessarily an expense, it might also be a transaction and in such case this link will be broken. See https://opencollective-frontend-xgxjzoyefm.now.sh/captainfact_io/v2 for example, donation is linked like an expense.

Also I'm not sure that we should provide the `description` here as it's already set on the child.

Reply:
Makes sense, fixed 👍